### DISCUSSION TITLE
Welcome to hsnips Discussions!

### DISCUSSION THREAD

**Discussion by Oskar-Idland** [Oskar-Idland/hsnips#5] [Announcements]
## 👋 Welcome! 
This is a place to:
  - Discuss new features 
  - Show off good snippets other might appreciate
  - Ask dev questions and to learn the codebase and contribute!

**Comment by jonas-frey** [comment]
Hi, what's this point of this fork? What can it do that dravin's version can't? I'm a great fan of the original, and wondering if I should swich. Is the original not maintained anymore?

**Reply by Oskar-Idland** [reply]
Yes, there was an issue which was not resolved regarding deletion of tab stops when nesting snippets. We tried contacting the creator for months with no response. We found the patch and I just forked and applied it. If the old one works for you, there is really no reason to switch. It's only when doing snippets in snippets you see the issue. I haven't implemented contexts yet, except for math mode. If this is a feature you need, you should stay for now. I will add it later. This version has the VISUAL variable which allows for embedding selected text inside a snippet. That's the only exclusive feature from the original fork, for now.
